rules/windows/sysmon/sysmon_powershell_without_powershell_exe.yml

```diff
@@ -0,0 +1,34 @@
+title: PowerShell without PowerShell.exe
+id: 3f07b9d1-2082-4c56-9277-613a621983cc
+description: Detects loading Powershell packet by non-Powershell process
+references:
+    - https://speakerdeck.com/heirhabarov/hunting-for-powershell-abuse
+tag:
+    - attack.defense_evasion
+status: experimental
+author: oscd.community, Natalia Shornikova
+date: 2020/10/06
+logsource:
+    product: windows
+    service: sysmon
+detection:
+    selection:
+      EventID: 7
+      ImageLoaded|endswith:
+        - '\System.Management.Automation.dll'
+        - '\System.Management.Automation.ni.dll'
+    filter:
+      SourceImage|endswith:
+        - '\powershell.exe'
+        - '\powershell_ise.exe'
+        - '\sqlps.exe'
+        - '\sdiagnhost.exe'
+        - '\wsmprovhost.exe'
+        - '\winrshost.exe'
+        - '\mscorsvw.exe'
+        - '\syncappvpublishingserver.exe'
+        - '\runscripthelper.exe'
+        - '\ServerManager.exe'
```

Comment:
Could you please explain why did you decide to not use commandline filters, that are present in the original rule? thank you!